src/assets/all_assets_request.rs

```diff
@@ -118,9 +90,15 @@ impl AllAssetsRequest {
     /// # Returns
     /// The request object
     /// [AllAssetsRequest](struct.AllAssetsRequest.html)
-    pub fn set_asset_issuer(&mut self, asset_issuer: &str) -> &mut Self {
-        self.asset_issuer = Some(asset_issuer.to_owned());
-        self
+    pub fn set_asset_issuer(self, asset_issuer: &str) -> Result<AllAssetsRequest, String> {
```

Comment:
We should add the boundaries to the documentation, so the developers know if they mess up, why their code doesn't compile.

Reply:
Moved comment to https://github.com/Baseflow/stellar-rust-sdk/issues/4